system/MockBox.cfc

```diff
@@ -371,7 +371,7 @@ Description		:
 				fncMD["access"] = "public";
 			}
 			if( not structKeyExists(fncMD,"output") ){
-				fncMD["output"] = false;
+				fncMD["output"] = true;
```

Comment:
I thought we wanted the default to be false?

Reply:
No, you're confused just like I was.  The default output is ALWAYS true on all CF engines regardless of tags or script.  Just because output is true, doesn't mean it will leak whitespace, it just means it can output if it needs to.  The correct default for output on a method is always true, the fact that it's now in script means, we don't care.